### PR TITLE
New Otsecurity Pack

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Minimum version of Cortex XSOAR
- [ x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ x] No

## Must have
- [ ] Tests
- [ x] Documentation

### CODE REVIEW

**Comment by dansterenson** [comment]
@ayman-m Thanks for your contribution. Can you please make sure the ci/circleci: Run Validations check passes? You can see the specific errors when pressing on "Details".

**Review by dansterenson — CHANGES_REQUESTED** [review]
@ayman-m Well done! please see my minor changes. @evisochek Can you please go over the playbook, layout, incidents type etc?

File: Packs/OTSecurity/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "OT Security",
```
[the remark this answers is not in the document]
Comment (ayman-m) [reply Packs/OTSecurity/pack_metadata.json]: @dansterenson , thank you, I have pushed a new commit to fix that

File: Packs/OTSecurity/IndicatorTypes/MAC_Address.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	"details": "MAC Address",
+	"disabled": false,
+    "shouldPublish": true,
+    "shouldCommit": true,
+    "expiration": 0,
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "system": false,
+	"id": "MAC Address"
+}
```
Comment (evisochek) [inline Packs/OTSecurity/IndicatorTypes/MAC_Address.json:12]: I think that indicator type should be associated to layout

Comment (ayman-m) [reply Packs/OTSecurity/IndicatorTypes/MAC_Address.json:13]: @evisochek I added a MAC Address layout

**Review by dansterenson — APPROVED** [review]

**Pull request merged**